src/DatePicker/DatePicker.js

```diff
@@ -150,11 +151,6 @@ class DatePicker extends React.Component {
     value: React.PropTypes.any,
 
     /**
-     * Creates a ValueLink with the value of date picker.
-     */
-    valueLink: React.PropTypes.object,
-
-    /**
      * Wordings used inside the button of the dialog.
      */
     wordings: deprecated(React.PropTypes.object, 'Instead, use `cancelLabel` and `okLabel`.'),
```

Comment:
@mbrookes Oh. Frack. No idea what the plan is. There's a lot in here but it also fixes a lot of bugs and doesn't actually have any API changes right? If that's the case it can go in any release.